packages/cli/templates/vue3/src/components/FieldPluginExample/index.vue

```diff
@@ -37,15 +37,15 @@ const plugin = useFieldPlugin({
     <div class="container">
       <Counter
         :count="plugin.data.content"
-        @onIncrease="plugin.actions.setContent(plugin.data.content + 1)"
+        @on-increase="plugin.actions.setContent(plugin.data.content + 1)"
       />
       <hr />
       <ModalToggle
-        :isModalOpen="plugin.data.isModalOpen"
-        :setModalOpen="plugin.actions.setModalOpen"
+        :is-modal-open="plugin.data.isModalOpen"
+        :set-modal-open="plugin.actions.setModalOpen"
       />
       <hr />
```

Comment:
Seems that the checks are complaining about the closing tag 
```suggestion
      <hr>
```

Reply:
could you show me where you see the checks complaining about this?

Reply:
This was the one: ![Screenshot 2024-09-25 at 13 32 17](https://github.com/user-attachments/assets/8ec9a784-e985-41f7-a16e-fc303e9a5bdc), also in Vue 2 as the one @demetriusfeijoo commented